### PR TITLE
Fixed Toaster Configuration 

### LINKED ISSUE #197: Toaster Not Working in DailyCode
i have noticed that toaster is not properly configured in the app, i have found the mistake and want to work on it

### PR DESCRIPTION
**Pull request by piyushhsainii**
closes #197

- Properly Exported toaster.tsx in packagae.json of UI directory
- Fixed Toaster usage in layout.tsx

Screenshot (363)

### CODE REVIEW

File: apps/web/.env.example [hunk 1]
```diff
@@ -1,3 +1,11 @@
+GITHUB_ID=""
+GITHUB_SECRET=""
+GOOGLE_CLIENT_ID=""
```
Comment (hkirat) [inline apps/web/.env.example:3]: Can u explain these changeS?

**Review by hkirat — CHANGES_REQUESTED** [review]
Is this PR up to date? I see a bunch of changes unrelated to the PR

File: apps/web/.env.example (hunk 1 above)
Comment (piyushhsainii) [reply apps/web/.env.example:3]: please check my other commit in this PR, this one i committed bymistake

Comment (hkirat) [reply apps/web/.env.example:3]: You'll have to remove that commit ser
Closing the PR for now, please re-open after the commits are fixed

**Pull request closed without merging**